integration/bin/tachyon-yarn-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# prepare Tachyon dir structure for a YARN container to launch master or worker
+tar zxf tachyon.tar.gz
+
+# launch master or worker on this container
+if [ $1 = 'master' ]; then
+  ./integration/bin/tachyon-master-yarn.sh
```

Comment:
use "${TACHYON_HOME} as an anchor instead of a relative path (that way the execution of the script does not depend on the current working directory)